src/bio2bel/sources/intact.py

```diff
@@ -160,28 +158,37 @@
         namespace='GO',
         identifier='0018377',
     ),
+    # modified
```

Comment:
i don't get it, why do we need a code comment to say that this is part of the diff? please remove

Reply:
I just wanted to mark it as a modified line, to check it up, but I can remove it